SourceX/qol.cpp

```diff
@@ -10,10 +10,25 @@
 namespace devilution {
 namespace {
 
+namespace xpbar {
+	constexpr int barWidth = 307;
+	constexpr int barHeight = 5;
+
+	constexpr int goldGradient[] = { 0xCF, 0xCE, 0xCD, 0xCC, 0xCB, 0xCA, 0xC9, 0xC8, 0xC7, 0xC6, 0xC5, 0xC4, 0xC3, 0xC2, 0xC1, 0xC0 };
+	constexpr int goldGrades = 12;

```

Comment:
There are about a dozen ways to skin this cat.

I don't really want to change the simplicity of the source code just to get some fancy container in there. The gradient array is actually longer than the number of colors I'm using (otherwise it would have been too yellow/too bright), that's why I have an explicit second variable that limits it to the darker 12 colors.